src/mem/ruby/protocol/chi/tlm/SConscript

```diff
@@ -69,13 +69,19 @@ if env["CONF"]["BUILD_TLM"]:
         sim_objects=['TlmGenerator'],
         tags=['arm isa']
     )
+    SimObject(
+        "TlmPort.py",
+        sim_objects=[],
+        tags=['arm isa']
```

Comment:
This doesn't need to be tagged Arm. It's pure python and has no dependencies on Arm ISA.

(Actually, that's probably true of most of the code in here. But it's OK to keep the other code as is. See below on this file anyway.)

Reply:
+1